poetry/utils/shell.py

```diff
@@ -9,6 +9,7 @@
 from shellingham import detect_shell
 
 from ._compat import WINDOWS
+from ._compat import Path
```

Comment:
Should that be merged with the previous line?

Reply:
poetry's isort configuration is preferring keeping this separated.